health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppFieldComposer.js

```
@@ -0,0 +1,1140 @@
+                      marginTop: "1rem",
+                    }}
+                  >
+                    <DustbinIcon />
+                  </div>
+                )}
+              </>
+            )}
+          </LabelFieldPair>
+        </>
+      );
+      break;
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)